src/sleep.cc

```diff
@@ -41,7 +41,7 @@ void SleepForMicroseconds(int microseconds) {
 }
 
 void SleepForMilliseconds(int milliseconds) {
-  SleepForMicroseconds(static_cast<int>(milliseconds) * kNumMicrosPerMilli);
+  SleepForMicroseconds(static_cast<int>(milliseconds * kNumMicrosPerMilli));
```

Comment:
Ah sorry I should have figured that out from #354. Go ahead and remove this change. I'm going to commit a different fix upstream. 

I think the actual bug here is that `kNumMicrosPerMilli` is specified as `int64_t`